queries.sql

```diff
@@ -24,4 +24,58 @@ WHERE NOT name ='Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg
 SELECT *
 FROM animals 
-WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is NULL;
+-- Commit the transaction.
```

Comment:
Please verify that change was made before `COMMIT`.
Please verify that change persists after `COMMIT`.